src/index.js

```diff
@@ -127,6 +127,14 @@ app.setAppUserModelId('com.marshallofsound.gpmdp.core');
 
     // and load the index.html of the app.
     mainWindow.loadURL(`file://${__dirname}/public_html/index.html`);
+
+    // minimize if 'start minimized' is on.
```

Comment:
I completed these changes and could submit them into a new PR or @snhardin could pull them himself to keep this PR going.

See: [zevisert/GPMDP/commit/`@a9dc220`](https://github.com/zevisert/Google-Play-Music-Desktop-Player-UNOFFICIAL-/commit/a9dc22093fecc32635266e1a6556f7bfd411c8eb)